scripts/run-vscode-integration-tests-with-top-level-extensions.js

```diff
@@ -8,23 +8,24 @@ const path = require('path');
 const cwd = process.cwd();
 
 // Executes the test, using the top-level packages as the CODE_EXTENSIONS_PATH
-
-shell.exec(
-  `cross-env CODE_EXTENSIONS_PATH='${path.join(
-    __dirname,
-    '..',
-    'packages'
-  )}' CODE_TESTS_WORKSPACE='${path.join(
+try {
+  const CODE_EXTENSIONS_PATH = path.join(__dirname, '..', 'packages');
+  const CODE_TESTS_WORKSPACE = path.join(
     __dirname,
     '..',
     'packages',
     'system-tests',
     'assets',
     'sfdx-simple'
-  )}' CODE_TESTS_PATH='${path.join(
-    cwd,
-    'out',
-    'test',
-    'vscode-integration'
-  )}' node ./node_modules/vscode/bin/test`
-);
+  );
+  const CODE_TESTS_PATH = path.join(cwd, 'out', 'test', 'vscode-integration');
+
+  const CODE_VERSION = '1.41.1';
```

Comment:
Why are we hardcoding the version of vscode to download and run tests with ? Right now we always download the latest stable version. In the past we've hardcoded the vscode version only when the stable version has regressions that cause the tests to fail and remove it once vscode addresses the issue.

Reply:
yeah were working through how to fix this still -- Its temporary but good catch